python/ray/experimental/serve/constants.py

```diff
@@ -1,2 +1,17 @@
 #: The interval which http server refreshes its routing table
 HTTP_ROUTER_CHECKER_INTERVAL_S = 2
+
+#: Actor name used to register actor nursery
+SERVE_NURSERY_NAME = "SERVE_ACTOR_NURSERY"
+
+#: KVStore connector key in bootstrap config
+BOOTSTRAP_KV_STORE_CONN_KEY = "kv_store_connector"
+
+#: HTTP Address
+DEFAULT_HTTP_ADDRESS = "http://0.0.0.0:8000"
```

Comment:
This will be addressed in next PR, when adding K8s support, we will start a HTTP server *per node*.